app/models/food.rb

```diff
@@ -0,0 +1,7 @@
+class Food < ApplicationRecord
+  belongs_to :user
+  has_many :recipe_foods, dependent: :destroy
+  has_many :recipes, through: :recipe_foods
+  validates :food, :measurement_unit, :price, :quantity, presence: true
```

Comment:
- [ ] Hi @benardop and @omarbabou, you cannot create food because of validation here. This is a Food model yet you are validating the presence of `food` here. This raises the error. When I removed the `food` from this line, I was able to create food from the controller.

  ```
  validates :measurement_unit, :price, :quantity, presence: true
  ```

Reply:
@TedLivist Thank you very much for your detailed CR feedback. We really appreciate it.